test/server.test.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
```

Comment:
Do we have tests that cover?:

* string port number
* port number like '1234garbage'

Reply:
Unfortunately not. I'll add them asap. Thanks for the review!

Reply:
I just updated the tests. Can you review again @jsumners?